subjects/byebyefirst/README.md

```diff
@@ -0,0 +1,46 @@
+## byebyefirst
+
+### Instructions
+
+Write a function that takes a slice of `string` and returns a new slice without the first element. 
```

Comment:
Should be slice of `strings`---> plural.

Reply:
fixed